replace cache diagrams (#5523)
dbt-labs/docs.getdbt.com: website/docs/docs/build/metrics-overview.md

| Parameter | Description | Type |
| --------- | ----------- | ---- |

> | `name` | Provide the reference name for the metric. This name must be unique amongst all metrics and can include lowercase letters, numbers, and underscores.  | Required |

Reviewer: ```suggestion
| `name` | Provide the reference name for the metric. This name must be a unique metric name and can consist of lowercase letters, numbers, and underscores.  | Required |
```
